src/i_video.c

```diff
@@ -315,16 +315,7 @@ static void AdjustWindowSize(void)
 {
     if (aspect_ratio_correct || integer_scaling)
     {
-        if (window_width * actualheight <= window_height * SCREENWIDTH)
-        {
-            // We round up window_height if the ratio is not exact; this leaves
-            // the result stable.
-            window_height = (window_width * actualheight + SCREENWIDTH - 1) / SCREENWIDTH;
-        }
-        else
-        {
-            window_width = window_height * SCREENWIDTH / actualheight;
-        }
+        window_width = window_height * SCREENWIDTH / actualheight;
```

Comment:
Also here, please avoid whitespace changes relative to the Choco code base by just keeping indentation - even if this makes it look inconsistent.